docs/content/guides/limiting-concurrency-in-data-pipelines.mdx

```diff
@@ -422,7 +422,7 @@ The concurrency key should match the name that the op/asset is tagged with. For
 
 A default concurrency limit can be configured for the instance, for any concurrency keys that do not have an explicit limit set:
 
-- **Dagster+**: Use the [Dagster+ UI or the `dagster-cloud` CLI][cloud-deployment-settings]
+- **Dagster+**: Use the deployment settings via the [Dagster+ UI or the `dagster-cloud` CLI][cloud-deployment-settings]
```

Comment:
"Use the deployment settings"  reads a little odd to me. Maybe "Edit the `concurrency` key in deployment settings via the`

Reply:
Yeah, this reads better to me.